src/components/DayInfo/DayInfo.js

```diff
@@ -4,7 +4,13 @@ import { convertKelvinToCelsius } from '../../utils/index';
 import classes from './DayInfo.module.scss';
 import { momentFormat } from '../../utils/index';
 
-export const DayInfo = ({ weather }) => {
+export const DayInfo = ({ weather, temp }) => {
+  let toggleKelvinCelsius = function () {
+    return temp
+      ? temp + (temp > 100 ? ' K' : ' \u2103')
```

Comment:
Could you explain the logic here? What does 'temp' mean in this case?

Reply:
'temp ' it  is our temperature from "const [temperature, setTemperature] = React.useState(null);";
if we have such prop we check is it Kelvin or Celsious to manage 'K" and 'C'. but maybe it's better to create new key in state.